Assets/Scripts/Scanner/Scanner.cs

```diff
@@ -51,6 +51,11 @@ private set
 		private Action<string, string> Callback;
 		private ParserResult Result;
 
+		//
+		private int delayFrameWebcamMin = 3;
```

Comment:
Hum don't really like a const here, it looks more like an option and should be customizable